react/src/components/molecules/Link/index.js

```diff
@@ -6,12 +6,13 @@ import { Icon } from '../../../index';
 
 const Link = (props) => {
   const icon = props.icon ? (<Icon name={props.icon} svgWidth={13.2} svgHeight={13.2} />) : '';
+  const classes = (props.classes) ? `js-clickable-link ${props.classes}` : 'js-clickable-link';
   return(
     <a
       href={props.href}
-      className="js-clickable-link"
+      className={classes}
```

Comment:
```suggestion
      className={props.classes}
```
or maybe we use the classname lib here